lib/internal/cluster/RoundRobinHandle.js

```diff
@@ -0,0 +1,113 @@
+'use strict';
```

Comment:
I think that BufferList is an unfortunate outlier. lib/internal/streams/lazy_transform.js is a better model to follow.